pymatviz/ptable/ptable_matplotlib.py

```diff
@@ -200,7 +200,7 @@ def ptable_heatmap(
             stacklevel=2,
         )
 
-    # Handle deprecated args  # TODO: remove after 2025-01-01
+    # Handle deprecated args  # TODO: remove after 2025-04-01
```

Comment:
Here I extended the deprecation deadline by 3 months in case someone might be filtering `UserWarning` (I used the incorrect UserWarning instead of DeprecationWarning)